dali/daliadmin/daadmin.hpp

```diff
@@ -28,8 +28,9 @@ extern DALIADMIN_API void xmlSize(const char *filename, double pc);
 extern DALIADMIN_API void translateToXpath(const char *logicalfile, DfsXmlBranchKind tailType = DXB_File);
 
 extern DALIADMIN_API void _export_(const char *path, const char *dst, bool safe = false);
-extern DALIADMIN_API void import(const char *path, const char *src, bool add);
-extern DALIADMIN_API void _delete_(const char *path, bool backup);
+extern DALIADMIN_API bool import(const char *path, const char *src, bool add, StringBuffer &out);
+extern DALIADMIN_API bool importXML(const char *path, const char *xml, bool add, StringBuffer &out);
+extern DALIADMIN_API bool _delete_(const char *path, bool backup,StringBuffer &out);
```

Comment:
we shouldn't have method names like this in headers, I see it was there from last PR, but we should change now.
A further improvement, since "delete" as a method name, at a global symbol name is unhelpful, would be to put all these daadmin methods in their own namespace, and refer to explicitly, e.g. daadmin::delete(...)

Reply:
Will rename it to delete() and add namespace daadmin.